css/styles.css

```diff
@@ -34,15 +35,15 @@ header, footer {
     width: 100%;
     display: none;
 }
-#playerControls {
-    display: none;
-}
-
 #zen-video-title #demo {
     margin-top: 5px;
     font-weight: bold;
     word-wrap: break-word;
 }
+#demo {
```

Comment:
Where did these numbers come from?
I'd like to see if we can do this without these seemingly random numbers